modules/openfast-library/src/FAST_Registry.txt

```diff
@@ -420,76 +424,101 @@ typedef	^	^	BD_MiscVarType	m	{:}	-	-	"Misc/optimization variables"
 typedef	^	^	BD_OutputType	Output	{:}{:}	-	-	"Array of outputs associated with CalcSteady Azimuths"
 typedef	^	^	BD_OutputType	y_interp	{:}	-	-	"interpolated system outputs for CalcSteady"
 typedef	^	^	BD_InputType	Input	{:}{:}	-	-	"Array of inputs associated with InputTimes"
+typedef	^	^	BD_InputType	Input_Saved	{:}{:}	-	-	"Backup Array of inputs associated with InputTimes"
 typedef	^	^	DbKi	InputTimes	{:}{:}	-	-	"Array of times associated with Input Array"
+typedef	^	^	DbKi	InputTimes_Saved	{:}{:}	-	-	"Backup Array of times associated with Input Array"
 
 # ..... ElastoDyn data .......................................................................................................
-typedef	FAST	ElastoDyn_Data	ED_ContinuousStateType	x	{2}	-	-	"Continuous states"
-typedef	^	^	ED_DiscreteStateType	xd	{2}	-	-	"Discrete states"
-typedef	^	^	ED_ConstraintStateType	z	{2}	-	-	"Constraint states"
-typedef	^	^	ED_OtherStateType	OtherSt	{2}	-	-	"Other states"
+typedef	FAST	ElastoDyn_Data	ED_ContinuousStateType	x	{4}	-	-	"Continuous states"
```

Comment:
I really wish we had defined a NumStates parameter, so the state arrays would be the same size across modules. Something to remember for the future.

Reply:
That's worth doing. I'll make the change.

Reply:
`NumStateTimes`